# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ooo/adjust_patches.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some patches are on apply list but not present on the release
Alejandro

--- ./patches/src680-m86/apply.orig	2005-03-16 00:24:44.000000000 -0400
+++ ./patches/src680-m86/apply	2005-03-18 02:02:33.404939104 -0400
@@ -177,20 +177,14 @@
 buildfix-rpm-4.4.diff
 # Fix for partial libwpd filter removal n#72313, i#45007
 crash-iodetect.diff
-# Make it compile even with visibility-enabled gcc
-buildfix-comphelper-visibility.diff
 # Fix a silly with multiple linkoo invocations i#45071
 linkoo-evil.diff
-# Fix XML slide export so we don't get problems on re-load in OO.o ... n#71105, i#45092
-sd-interop-1.1.diff
 
 [ LinuxOnly ]
 # Optimize linkage of some xmlhelp libraries # FIXME: shres
 xmlhelp-optimize-linkage.diff
 # Modify the fpicker makefile causing problems in the win32 build
 fpicker-common-build.diff
-# fix for the XHTML export not working against the older temporary fix n#66586
-fpicker-csv-filter-open-fix.diff
 
 [ XineramaPIC ]
 # use libXinerama_pic.a on all archs
@@ -322,8 +316,6 @@
 odk-java-wizpanel.diff
 # add possibility to set ARCH_FLAGS when building libnas
 nas-use-arch-flags.diff
-# enable LDAP backend installation
-scp-ldapbe2.diff
 # fix system-portaudio and system-sndfile
 vcl-pasf-system-buildfix.diff
 
@@ -406,8 +398,6 @@
 fpicker-fixes.diff
 # fix for the gtk critical error n#66349
 fpicker-gnome-crash-fix.diff
-# temporary fix to enable filter detection to work properly i#10545
-sfx2-fpicker-gnome-filter-fix.diff
 # fix for the fpicker hanging on accessible desktop
 fpicker-accessibility-fix.diff
 # fix for the fpicker crashing in the Euro converter n#67124
